addons/midi/addon.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "MIDI output",
+  "description": "Connect Music blocks to a MIDI device.",
```

Comment:
Unlike the "gamepad support" addon, many of our users (myself included) are probably not familiar with MIDI devices. Maybe we can mention an example of what can be done with the addon here, and how to know if a device is compatible.